src/main/resources/templates/index.html

```diff
@@ -3,13 +3,17 @@
 <head>
 <meta charset="ISO-8859-1"/>
 <!-- Latest compiled and minified CSS -->
-<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css">
+<!--<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css">-->
```

Comment:
Don't comment out code and commit it... that's a version control no-no, and it's clutter.  You can always look back in history to see what used to be there.  Besides, you're in your own branch, which is a snapshot of master... so you can do what you want.